src/RiotAPILibraries.ts

```diff
@@ -84,76 +84,95 @@ export default class RiotAPILibraries {
         if (this.settings.githubLibraries.aliases.some(x => x === command)) {
 
             if (language === "list") {
-                const responses = await fetch(this.settings.githubLibraries.baseURL, this.fetchSettings);
-                const data = await responses.json() as GithubAPIStruct[];
-
-                const languages = "`" + data.map(x => x.name).join(", ") + "`";
-                const reply = this.settings.githubLibraries.languageList.replace("{languages}", languages);
-                message.channel.send(reply);
-                return;
-            }
+                try {
+                    const responses = await fetch(this.settings.githubLibraries.baseURL, this.fetchSettings);
+                    const data = await responses.json() as GithubAPIStruct[];
+
+                    const languages = "`" + data.map(x => x.name).join(", ") + "`";
+                    const reply = this.settings.githubLibraries.languageList.replace("{languages}", languages);
+                    message.channel.send(reply);
+                } catch (error) {
+                    console.error(`Error occurred while fetching from github: ${error}`);
```

Comment:
Maybe send a message to the user, so they aren't left waiting for nothing?

Reply:
Fixed, github wont show as outdated for some reason..